modules/nbde-automatic-start-at-boot.adoc

```diff
@@ -0,0 +1,10 @@
+// Module included in the following assemblies:
+//
+// scalability_and_performance/ztp-nbde-implementation-guide.adoc
+
+[id="ztp-nbde-automatic-start-at-boot_{context}"]
```

Comment:
I'm still seeing `ztp-` in comments and ID tags; should these be renamed so everything matches?